src/mongooseim.app.src

```diff
@@ -51,7 +51,8 @@
                   gen_fsm_compat,
                   erl_csv_generator,
                   trails,
-                  cowboy_swagger
+                  cowboy_swagger,
+                  flatlog
```

Comment:
Why do we need to add it here? There's no application to start 🤔

Reply:
It's an easy way to put it into a release.